torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -854,22 +879,35 @@ NvrtcFunction nvrtcCompile(
           options.size(),
           options.data(),
           option_vals.data()));
+
+      if (isDebugDumpEnabled(DebugDumpOption::PrintPtxasLog)) {
+	printf("%s\n", info_log.data());
```

Comment:
Do we have to use `printf`? I remember from a long time ago that upstream doesn't like it. Not sure if it's still valid:

[https://github.com/pytorch/pytorch/pull/46994](https://github.com/pytorch/pytorch/pull/46994)